### PR TITLE
Fixes #9483 - notify the services after gutterball configuration changes

### PR DESCRIPTION
**Pull request by iNecas**

### CODE REVIEW

File: manifests/plugin/gutterball.pp [hunk 1]
```diff
@@ -13,5 +13,6 @@
     foreman_plugins_dir => $katello::config_dir,
     foreman_user        => $katello::user,
     foreman_group       => $katello::group,
+    notify              => [Service['foreman-tasks'], Service['foreman']],
```
Comment (ekohl) [inline manifests/plugin/gutterball.pp]: Often the foreman service is run through apache. It's be better to notify ```Class['foreman::service']``` which does the right thing for you.

Comment (iNecas) [inline manifests/plugin/gutterball.pp]: Good point, fixed

**Comment by ekohl** [comment]
:+1: but tests are broken because puppet-pulp dropped puppet 2.7 support. @ehelms I think you want to remove puppet 2.7 from the test matrix.

**Comment by ehelms** [comment]
Removing 2.7 support - https://github.com/Katello/puppet-katello/pull/63

**Comment by ekohl** [comment]
@iNecas can you rebase now #63 is merged?

**Comment by iNecas** [comment]
Rebased

**Comment by ekohl** [comment]
:+1: yay green tests

**Comment by ehelms** [comment]
ACK

**Pull request merged**